### ISSUE TITLE
Better handling of JsonRPC logs

### ISSUE THREAD

**Issue by MarekM25** [NethermindEth/nethermind#5756]
CL can send tons of logs, and as an effect, we have logs like on the below screen.
![Screenshot from 2023-06-02 17-04-16](https://github.com/NethermindEth/nethermind/assets/9356351/6c73e1bb-200b-43f0-81f1-20c09c8970b1)

It would be better to filter them.

An alternative solution would be add a config option. We could introduce a new setting: JsonRpc.Logs with the following values (just an example):
```
None (future default option nr 1)
EnginePortRequest
EnginePortRequestAndResponse
NonEnginePortsRequest (future default option nr 2)
NonEnginePortsRequestAndResponse
AllRequest (current option)
All

```
It might be useful for debugging (for example hive tests). Of course, we can configure JsonRpc logs in Nlog, but it is not convenient enough

**Comment by eliotstock** [comment]
I agree these are too noisy for `INFO` level, but I'd challenge whether anyone wants the added complexity of a new config option. It's simpler just to move most of these to `DEBUG` level and let anyone who wants to see them change log levels, no?

Not quite sure what you mean by "Of course, we can configure JsonRpc logs in Nlog, but it is not convenient enough".

**Comment by kamilchodola** [comment]
@MarekM25 @eliotstock We already moved all rpc calls to debug and left only warnings when some of the rpcs are failing on the node - i think then this is to be closed.

**Comment by svlachakis** [comment]
Seems outdated, feel free to reopen if it's still an issue
